.travis.yml

```diff
@@ -16,14 +16,14 @@ addons:
 matrix:
```

Comment:
We should probably remove this file.

Reply:
It's very convenient to be able to run my own changes in CI. Travis works fine for that purpose, I don't know if Azure offers anything comparable. At least CircleCI didn't work well at all for me.